doc/classes/AnimationNodeOneShot.xml

```diff
@@ -35,10 +35,10 @@
 		[/gdscript]
 		[csharp]
 		// Play child animation connected to "shot" port.
-		animationTree.Set("parameters/OneShot/request", AnimationNodeOneShot.ONE_SHOT_REQUEST_FIRE);
+		animationTree.Set("parameters/OneShot/request", (int)AnimationNodeOneShot.OneShotRequest.Fire);
 
 		// Abort child animation connected to "shot" port.
-		animationTree.Set("parameters/OneShot/request", AnimationNodeOneShot.ONE_SHOT_REQUEST_ABORT);
+		animationTree.Set("parameters/OneShot/request", (int)AnimationNodeOneShot.OneShotRequest.Abort);
 
 		// Abort child animation with fading out connected to "shot" port.
 		animationTree.Set("parameters/OneShot/request", AnimationNodeOneShot.ONE_SHOT_REQUEST_FADE_OUT);
```

Comment:
Needs the same change once more.

Reply:
It's `AnimationNodeOneShot.OneShotRequest.FadeOut`, it was added by https://github.com/godotengine/godot/pull/74190.